mlir/test/Conversion/ArithToEmitC/arith-to-emitc-unsupported.mlir

```diff
@@ -16,39 +16,6 @@ func.func @arith_cast_vector(%arg0: vector<5xf32>) -> vector<5xi32> {
 
 // -----
 
-func.func @arith_cast_bf16(%arg0: bf16) -> i32 {
```

Comment:
Can you keep the tests and replace the types with unsupported floating point types, like `f128` and `f80` (and `tf32` is a thing as well, I think)

Reply:
Done